lib/bap_types/bap_bytes.ml

```diff
@@ -0,0 +1,83 @@
+
+module Std_bytes = Bytes
+
+open Core_kernel.Std
+
+module Binable = Bin_prot.Utils.Make_binable(struct
+    module Binable = String
+    type t = Std_bytes.t
+    let to_binable = Std_bytes.unsafe_to_string
+    let of_binable = Std_bytes.of_string 
+  end)
+
+module Stringable = struct
+  type t = Std_bytes.t
+  let of_string = Std_bytes.of_string
+  let to_string = Std_bytes.unsafe_to_string
+end
+
+module Sexpable = Sexpable.Of_stringable(Stringable)
+
+module T = struct
+  include Std_bytes
+  include Binable
+  include Sexpable
+end
+
+module Elt = struct
+  type t = char
+  let equal t t' = Char.compare t t' = 0
+  let of_bool b = if b then 'a' else 'b'
+end
+
+module Seq = struct
+  type elt = Elt.t
+  type t = T.t with sexp
+
+  let create ~len = T.create len
+  let length = T.length
+  let get = T.get
+  let set = T.set
+  let unsafe_blit ~src ~src_pos ~dst ~dst_pos ~len = 
+    Std_bytes.blit src src_pos dst dst_pos len
+end
+
+module Blit = Blit.Make(Elt)(Seq)
+
+module Identifiable = Identifiable.Make(struct
+    type t = T.t with bin_io, compare, sexp
+    let of_string = Stringable.of_string
+    let to_string = Stringable.to_string
+    let hash = Hashtbl.hash
+    let module_name = "Bap bytes"
```

Comment:
it should be `Bap.Std.Bytes`. The module name is used to register printers in a toplevel. See `Printable` and `Install_printers` modules.